actions.go

```diff
@@ -3,10 +3,258 @@ package neo4j
 import (
 	"fmt"
 
+	"github.com/cloudprivacylabs/lsa/pkg/ls"
 	"github.com/cloudprivacylabs/opencypher/graph"
+	"github.com/fatih/structs"
 	"github.com/neo4j/neo4j-go-driver/v4/neo4j"
 )
 
+type neo4jAction interface {
+	Queue(neo4j.Transaction) error
```

Comment:
For queue, also pass a JobQueue, so you can actually queue the new job:

Queue(neo4j.Transaction,*JobQueue) error